ci.rb

```diff
@@ -183,7 +183,7 @@ def get_limits(t_options, t_client, t_repo)
     test_mode = !(ARGV[0] =~ /false/i)
 
     $logger.info "Querying for updated branches"
-    b.query_releases
+    # b.query_releases  releases are handled with github actions for now
```

Comment:
This is actually the only change I wanted to make today.  But it led to lots of rubocop changes.  This simply **doesn't** query for new release tags since we are building them on Github Actions now instead.